components/fonts/font.rs

```diff
@@ -66,11 +66,13 @@ pub trait PlatformFontMethods: Sized {
     fn new_from_template(
         template: FontTemplateRef,
         pt_size: Option<Au>,
+        data: &Arc<Vec<u8>>,
```

Comment:
By the way I don't understand the need for `Arc<Vec<u8>>` to be ret, why not just an `Rc<>`? (unlike the shared mem, the vec is not send or sync so I see no need for an arc).

Reply:
This could ultimately be `&[u8]`, I think.

Reply:
Ah, I was wrong sorry. It seems the reason is that the Windows platform font handle takes an `Arc`: https://docs.rs/dwrote/latest/dwrote/struct.FontFile.html#method.new_from_data

Reply:
Ok, thanks again for the info.